### PR TITLE
freecad 0.21 clean

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ requirements:
         - pivy
         - pyside2
         - smesh
-        - netgen 6.2.1808*
+        - netgen 6.2.2203*
```
Comment (adrianinsaval) [inline conda/meta.yaml:56]: @looooo are these conda files even used? I think everything is now on the freecad-feedstock repo isn't it?

File: .project [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adrianinsaval) [inline .project:1]: What's the purpose of this file?

**Comment by luzpaz** [comment]
Just want to keep in mind that there is an interest in US keeping it simple for realthunder and other toponaming devs to work on the big merge. It is potentially harder for them to do it when we make all these different changes (some of them cosmetic). Maybe worth discussing?

File: .project (hunk 2 above)
Comment (mosfet80) [reply .project:1]: it was added automatically by Eclipse .. it is not necessary

Comment (adrianinsaval) [reply .project:1]: IMO better to just remove it then, consider adding this to gitignore

File: conda/meta.yaml (hunk 1 above)
Comment (adrianinsaval) [reply conda/meta.yaml:56]: IMO better to remove it, if someone uses it they'll ask for it and we can direct them to the actually maintained files

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7022">pipeline status</a> for feature branch [PR_7022](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7022). Pipeline [563630680](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/563630680) was triggered at [75a8fa0](https://github.com/FreeCAD/FreeCAD/pull/7022/commits/75a8fa0b40394d370c133b3960652e52c06a1777). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request closed without merging**